src/routes/safe/components/Transactions/TxList/styled.tsx

```diff
@@ -469,6 +469,10 @@ export const OwnerListItem = styled.li`
 
 export const InlineEthHashInfo = styled(PrefixedEthHashInfo)`
   display: inline-flex;
+
+  span {
+    font-weight: normal;
```

Comment:
Why are you setting this here? Did you do so in `PrefixedEthHashInfo`? There is no programmatic control over it.

Reply:
In TxList we are using the inlined `PrefixedEthHashInfo` in front of bold text. By styling it here I want to unset the `shortName` boldness for `PrefixedInlineEthHashInfo`.
Should I pass a prop to address it programaticaly?

Reply:
Again, I thought you were modifying the wrapper directly. This is fine.